core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/model/upgrade/DistroXSyncCmV1Request.java

```diff
@@ -0,0 +1,29 @@
+package com.sequenceiq.distrox.api.v1.distrox.model.upgrade;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import com.sequenceiq.cloudbreak.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModelProperty;
+
+public class DistroXSyncCmV1Request {
+
+    @ApiModelProperty(ModelDescriptions.CmSyncRequest.IMAGE_IDS)
+    private Set<String> candidateImageUuids = new HashSet<>();
```

Comment:
why do we want to expose candidateImageUuids? It is only because of the additional time to run if we would omit it?
I am asking because 
- you've made some measures that looping across all images doesn't take that much time
- I find it difficult to imagine that any end user would call it with candidateImageUuids set.

Reply:
A not so straightforward usecase would be: 
upgrade fails and customer installs parcels by hand. But, for some reason he installs a different version of a parcel than stated in the package version on the target image. In this case cloudbreak would not sync the new version.

However, there could be other ways to do that, and maybe we should not be so restrictive.

Anyways, I removed it now.